src/index.html

```diff
@@ -1,13 +1,390 @@
 <!DOCTYPE html>
-<html lang="en">
+<html class="page" lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>MyBiKE</title>
+    <link
+      type="image/x-icon"
+      href="images/MyBiKE.svg"
+      rel="shortcut icon"
+    >
+    <link
+      type="Image/x-icon"
+      href="images/MyBiKE.svg"
+      rel="icon"
+    >
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body" id="home">
+    <header class="header">
```

Comment:
can you restrict header image size somehow? it looks weird with text on 1920 breakpoint
![image](https://user-images.githubusercontent.com/37161315/169851035-0496ecbb-6a9a-4ff5-af79-43c45e80be0b.png)

Reply:
Also header text color is wrong, go to figma and check for yourself